ocrd_models/ocrd_page_user_methods.py

```diff
@@ -113,6 +113,7 @@ def _add_method(class_re, method_name, file_name=None):
     _add_method(r'^(BorderType|RegionType|TextLineType|WordType|GlyphType)$', 'set_Coords'),
     _add_method(r'^(PageType)$', 'set_Border'),
     _add_method(r'^(CoordsType)$', 'set_points'),
+    _add_method(r'^(PageType)$', 'get_AllTextLine'),
```

Comment:
By analogy to our `get_AllRegions`, IMO this should also be plural here:

```suggestion
    _add_method(r'^(PageType)$', 'get_AllTextLines'),
```

Reply:
yep, it's also `get_AllAlternativeImages` now, I'll fix this.